.env.localhost

```diff
@@ -17,7 +17,7 @@ ENABLE_DEV_BACKEND_API="true"
 #FEATURE_APPLOCK_UNFOCUS_TIMEOUT="30"
 #FEATURE_APPLOCK_SCHEDULED_TIMEOUT="30"
 #FEATURE_ENABLE_MLS="true"
-#FEATURE_ENABLE_REACTION="false"
+#FEATURE_ENABLE_REACTION="true"
```

Comment:
Is this consumed anywhere? 
Maybe this can be removed altogether now that we are developing the feature entirely in this branch (we don't need to keep the simple `like` around if we go this strategy)

Reply:
yes, it is being used inside ContentMessage.tsx  
const isReactionFeatureEnabled = Config.getConfig().FEATURE.ENABLE_REACTION;

we need the feature flag to disable the feature from being displayed on dev, after we merge.